google/cloud/bigquery/_job_helpers.py

```diff
@@ -257,3 +287,48 @@ def do_query():
     future._job_retry = job_retry
 
     return future
+
+
+def query_and_wait(
+    client: "Client",
+    query: str,
+    job_config: Optional[job.QueryJobConfig],
+    location: Optional[str],
+    project: str,
+    retry: retries.Retry,
+    timeout: Optional[float],
+    job_retry: retries.Retry,
+) -> table.RowIterator:
+    """Initiate a query using jobs.query with jobCreationMode=JOB_CREATION_REQUIRED.
```

Comment:
Should this be JOB_CREATION_OPTIONAL?
It seems that way based on the descriptions above.

Reply:
You're correct. I'll update.